onnx/backend/test/case/node/stft.py

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class STFT(Base):
+
+    @staticmethod
+    def export() -> None:
+        signal = np.arange(0, 128, dtype=np.float32).reshape(1, 128, 1)
+        length = np.array((16))
```

Comment:
```suggestion
        length = np.array((16)).astype(np.int64)
```
NumPy will use numpy.int32 by default on Windows and use numpy.int64 by default on Mac. Specifying the variable type should resolve this issue.